src/assets/components/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function Button({buttonType, buttonText, clickHandler}) {
+    return (
+
+        <button
+            type={buttonType}
+            onClick={clickHandler}
+        >
+            {buttonText}
+        </button>
+
+    )
+}
```

Comment:
Dit zie ik nu als een doorgeefluik. Er wordt hier niets anders gedaan dan een ander component teruggeven en de props doorgegeven. Je had ook direct <button ... /> kunnen gebruiken en dan had je geen los component hoeven te maken. Mocht je bijv. een statisch classname setten, dan is het weer wél handig om een apart component te hebben, omdat je dat anders moet herhalen 😉